src/main/resources/auditing.yaml

```diff
@@ -11,4 +11,4 @@ fqns:
   PRINCIPAL: "ol.userid"
   TIMESTAMP: "general.datetime"
   OPERATION_ID: "ol.auditeventid"
-enabled: false
+enabled: true
```

Comment:
You can also just delete this property, bc it has a `true` default value

Reply:
Oh I forgot to change that back to false. I think dave added that field recently?

Reply:
Not sure about that.. @dbt ?

Reply:
I added it so people didn't keep commenting out the body of the audit method when running shuttle jobs, yeah :)